go/vt/mysqlctl/builtinbackupengine.go

```diff
@@ -579,22 +582,22 @@ func (be *BuiltinBackupEngine) restoreFile(ctx context.Context, cnf *Mycnf, bh b
 	var source io.ReadCloser
 	source, err = bh.ReadFile(ctx, name)
 	if err != nil {
-		return err
+		return vterrors.Wrap(err, "can't open source file for reading")
```

Comment:
might it be a good idea to include the file name in the returned error in all the places where we have added logging?

Reply:
I inject the source/dest file names uniformly one level up where `restoreFile` is called. Generally I prefer to inject params into errors at the caller side since the caller has more context on which arguments are important for what it's trying to do.

Reply:
+1